samples/deploy_model_sample.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START aiplatform_deploy_model_sample]
+from google.cloud import aiplatform
+
+
+def deploy_model_sample(
+    project: str,
+    endpoint_id: str,
+    model_name: str,
+    deployed_model_display_name: str,
+    location: str = "us-central1",
+    timeout: int = 7200,
```

Comment:
does this timeout need to be this high?

Reply:
Same idea as the other comment - I'm going to create issues for these, for further discussion